profiles/minimal/nsswitch.conf

```diff
@@ -1,7 +1,8 @@
 # In order of likelihood of use to accelerate lookup.
 passwd:     files {if "with-altfiles":altfiles }systemd
-shadow:     files
+shadow:     files systemd
 group:      files {if "with-altfiles":altfiles }systemd
```

Comment:
hmm, this should really use `[SUCCESS=merge]` before `systemd`, so that that users provided by nss-systemd can also be part of groups defined by something else.

(but that's not really an issue with this PR i ugess, but a pre-existing issue.)

Reply:
i am not entirely sure how this actually should look like in the end, because if you have both `files` and `altfiles` enabled, then systemd might want to merge group member lists into records returned by either, but `altfiles` might not be read to merge stuff into `files` output?

Reply:
Hmm. I'll look into it, nobody asked for this yet though, so perhaps it is not that important.